docs/hazmat/primitives/asymmetric/utils.rst

```diff
@@ -28,3 +28,40 @@ Asymmetric Utilities
     :param int s: The raw signature value ``s``.
 
     :return bytes: The encoded signature.
+
+.. function:: encode_ec_point(key_length, x, y)
```

Comment:
Yeah - passing in curve here instead of directly passing key length is definitely preferable.